eo-maven-plugin/src/main/java/org/eolang/maven/PullMojo.java

```diff
@@ -92,34 +87,49 @@ public final class PullMojo extends SafeMojo {
      * -DofflineHash=0.2.7:abc2sd3,0.2.8:s4se2fe
      *
      * @checkstyle MemberNameCheck (7 lines)
+     * @checkstyle VisibilityModifierCheck (6 lines)
      */
     @Parameter(property = "offlineHash")
     private String offlineHash;
 
     /**
      * The objectionary.
+     *
+     * @checkstyle VisibilityModifierCheck (5 lines)
```

Comment:
@maxonfjvipon Same here. Could you please explain why we need to exclude the `VisibilityModifierCheck` here?